src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const activeImg = document.querySelector('.gallery__large-img');
+const thumb = document.querySelector('.gallery__list');
+
+thumb.addEventListener('click', (clickevent) => {
+  const link = clickevent.target.closest('.list-item__link');
+  const img = clickevent.target.closest('.gallery__thumb');
+
+  clickevent.preventDefault();
+
+  if (!link || !img) {
+    return;
+  }
+
+  const imgSrc = link.getAttribute('href');
```

Comment:
Consider checking if `imgSrc` is a valid URL before setting it to `activeImg`. This will prevent potential issues if the `href` attribute is missing or malformed.